index.html

```diff
@@ -0,0 +1,352 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Enhance Your Skills GLOBAL Summit 2022</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+    />
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav
+      class="navbar nav1 navbar-expand-sm bg-dark navbar-dark justify-content-end py-0 p-3"
+    >
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/linke.png" alt="linkedin" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">English</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">My Page</a>
+        </li>
+        <li class="nav-item navLogout">
+          <a class="nav-link" href="#">Logout</a>
+        </li>
+      </ul>
```

Comment:
- [ ] The original design uses a whit color for the icons in the top navbar, kindly do the sme to match it as much as possible

Your design
![image](https://user-images.githubusercontent.com/81932478/205464401-fe3d732c-504a-4584-bb23-f82c6681bec7.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/205464429-f7836d71-bd24-4f1b-90c3-c7c1d35a389f.png)

Reply:
Thanks for the suggestion. It is done other than icons